content/chat/rooms/reactions.textile

```diff
@@ -17,11 +18,16 @@ Room reactions are ephemeral and not stored or aggregated by Ably. The intention
 
 h2(#subscribe). Subscribe to room reactions
 
+Room reactions events are emitted whenever a user sends a reaction, such as by hitting a 'like' button or clicking a heart emoji.
+
 blang[javascript].
-  Subscribe to room reactions by registering a listener. These events are emitted whenever a user sends a reaction, such as by hitting a 'like' button or clicking a heart emoji. Use the "@reactions.subscribe()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat_js.RoomReactions.html#subscribe method in a room to receive reactions:
+  Subscribe to room reactions by registering a listener. Use the "@reactions.subscribe()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat_js.RoomReactions.html#subscribe method in a room to receive reactions:
 
 blang[react].
   Subscribe to room reactions with the "@useRoomReactions@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/functions/chat_react.useRoomReactions.html hook. Supply an optional listener to receive the room reactions.
+  
+blang[swift].
+  Subscribe to a room reactions by creating a subscription. Use the @occupancy.subscribe(bufferingPolicy: .unbounded)@ method in a room to receive reactions:
```

Comment:
Should be `reactions` here instead of `occupancy`

Reply:
9df13d765ba32ee7a0cbc4e463308a4e75b75d9e

Reply:
updated